packages/terra-compact-interactive-list/src/subcomponents/Row.module.scss

```diff
@@ -1,6 +1,6 @@
 // Themes
-@import '../orion-fusion-theme/CompactInteractiveList.module';
-@import '../clinical-lowlight-theme/CompactInteractiveList.module';
+@import '../clinical-lowlight-theme/Row.module';
+@import '../clinical-lowlight-theme/Row.module';
```

Comment:
Could this be a typo?  It looks like there are two of the same imports.

Reply:
Good catch, fixed: https://github.com/cerner/terra-framework/pull/1950/commits/4bd839ac7e43b2c77fb8d914072df0cf1ad3b8ee